04-single-hop-6lowpan-icmp/example_test_guide.md

```diff
@@ -0,0 +1,168 @@
+_The following is a possible way to run the tests, this is just an example_
+## Goal: Check single hop 6lowpan icmp
+
+_note: Since the ping6 function is now async some interval times may need to
+be increased to compensate_
```

Comment:
This could be removed, I think. Nobody will care about this in >3 years, when nobody remembers the old ping ;-).

Reply:
I think I would like to state that intervals may change.  This may happen in the future as I found different boards require different intervals for different sizes.  If you really want I can remove it (and I would also hope people update the docs within 3 years).

Reply:
>  This may happen in the future as I found different boards require different intervals for different sizes.

Many things may happen in the future. We should adapt the docs then and not preemptively state maybes that only confuse people ;-).

Reply:
Sure thing, will do.